src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -61,10 +56,7 @@ export default class DetailsBody extends React.Component<Props> {
       id: undefined,
       name: (incident && incident.title) || '',
       fields: ['issue', 'count(id)', 'count_unique(user.id)'],
-      orderby:
-        incident.alertRule.aggregation === AlertRuleAggregations.UNIQUE_USERS
-          ? '-count_unique_user_id'
-          : '-count_id',
+      orderby: `-${incident.alertRule.aggregate}`,
```

Comment:
Need confirmation that this is right and that i can do this. @wedamija. I feel it is.. not right

Reply:
It's right.

Reply:
Are you sure? this will be `-percentile(transaction.duration, 0.5)` that's right?

Reply:
Ah.

you'll need to run it through https://github.com/getsentry/sentry/blob/cc3a93b7675bca623475f2bb0ba8b56b6e4f6228/src/sentry/static/sentry/app/utils/discover/fields.tsx#L373-L383